### ISSUE TITLE
Any progress?

### ISSUE THREAD

**Issue by ThomasLohmann** [kube-vip/kube-vip-dns#2]
Hi @thebsdbox do you plan to finalize the connection between kube-vip and external-dns in best case with Cloudflare (most popular one for DNS)? Since DNS is a really important thing in every application. And to automate this part would avoid a lot of headaches/issues.

Kind regards,
Thomas

**Comment by thebsdbox** [comment]
Yes, apologies for this i've been mainly focussed on some other Bare Metal Kubernetes work. I'll try and circle back on this hopefully this week.
